templates/graphdb-provision-user-secret.yaml

```diff
@@ -0,0 +1,16 @@
+# Secret used from the jobs to authenticate to running GraphDB instances
+apiVersion: {{ .Values.versions.secret }}
+kind: Secret
+metadata:
+  name: graphdb-provision-user
+  labels:
+    {{- include "graphdb.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade, pre-rollback, post-install, post-upgrade, post-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation, hook-succeeded, hook-failed
+    "helm.sh/hook-weight": "-9"
+type: Opaque
+data:
+  provisioningUsername: {{ .Values.graphdb.security.provisioningUsername | b64enc | quote }}
+  provisioningPassword: {{ .Values.graphdb.security.provisioningPassword | b64enc | quote }}
+  PROVISION_USER_AUTH_TOKEN: {{ printf "%s:%s" .Values.graphdb.security.provisioningUsername .Values.graphdb.security.provisioningPassword | b64enc | b64enc | quote }}
```

Comment:
Double `| b64enc | b64enc` ?

Reply:
The token is b64enc by default, and another one for the secret. When requested from the env it will be only once as expected

Reply:
I see, I tought it was a copy/paste mistake but I confirm that it works.